base/testfiles-ltcmd/github-1009.tlg

```diff
@@ -31,6 +43,30 @@ l. ...}
 ->.
 <recently read> }
 l. ...}
+> \foo=document command:
+  #1:+m
+  #2:+m
+->.
+<recently read> }
+l. ...}
+> \baz=document command:
+  #1:+m
+  #2:+m
+->.
+<recently read> }
+l. ...}
+> \foo=document command:
+  #1:m
+  #2:+m
+->.
+<recently read> }
```

Comment:
`m +m` shouldn't be optimized.

Reply:
It's not: I added the test purely as things would go sideways if it was optimized accidentally.

Reply:
Ignore me. `\ShowCommand` works as expected here.